### PR TITLE
Add Sileo Depictions

### PR DESCRIPTION
**Pull request by CobreDev**
Hey man, I think I'm done adding Sileo depictions for you. After you accept this, check it out! If you need help doing anything, don't hesitate to let me know!

Thanks!

### CODE REVIEW

File: sileo-featured.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "class":"FeaturedBannersView",
+  "itemSize":"{263, 148}",
+  "itemCornerRadius":10,
+  "banners":
+   [
+     {
+       "url":"https://raw.githubusercontent.com/mac-user669/cydia/master/assets/sileobanners/LotusDarkExtra.png",
```
Comment (favna) [inline sileo-featured.json]: Could you edit the URLs to refer to my repo `/favna/cydia` instead of yours?

**Review by favna — CHANGES_REQUESTED** [review]
Just 1 remark that should be a find-and-replace

File: sileo-featured.json (hunk 1 above)
Comment (CobreDev) [reply sileo-featured.json]: *Facepalm* Sorry, hang on

Comment (CobreDev) [reply sileo-featured.json]: Done

**Comment by CobreDev** [comment]
Uhh, hang on. I forgot to modify Packages

**Comment by CobreDev** [comment]
There we go. You should be ready to go. If you wanted to check it out, add mac-user669.github.io/cydia to your Sileo and LMK what to change. Editing the description and changelog should be pretty self-explanatory.

**Review by favna — APPROVED** [review]

**Comment by favna** [comment]
There are a few things that I want to change but I won't bother you with those since I can do them myself for sure. Like you said, self explanatory. This'll for sure do for a nice basis though. TYVM!

**Comment by CobreDev** [comment]
Thanks!

**Pull request merged**